src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.module.scss

```diff
@@ -1,4 +1,7 @@
-a.attentionBoxLink {
+// TODO: use just a class (without [href]) after removing anchor style from Typography
+// Using [href] to increase specificity and override `Typography a` styles
+a[href].attentionBoxLink,
+a[href].attentionBoxLink:hover {
```

Comment:
Anchor style from `Typography` is overriding it so I had to make it more specific, also added a comment to remove anchor style from Typography as users can just use the Link component